app/models/grda_warehouse/lookups/coc_code.rb

```diff
@@ -51,8 +66,12 @@ def self.options_for_select(user:, permission: :can_view_projects)
 
   def as_select_option
     [
-      "#{preferred_name || official_name} (#{coc_code})",
+      name,
       coc_code,
     ]
   end
+
+  def name
+    "#{preferred_name || official_name} (#{coc_code})"
```

Comment:
do we need to handle blank strings?

```suggestion
      "#{preferred_name.presence || official_name.presence || 'Unknown'} (#{coc_code.presence || 'No CoC'})"
```

Reply:
Official name and CoC Code should never be blank, adding presence call on preferred_name to handle emtpy strings.